app/controllers/mixins/ems_common_angular.rb

```diff
@@ -134,7 +134,7 @@ def ems_form_fields
       keystone_v3_domain_id = ""
       hawkular_hostname = ""
       hawkular_api_port = ""
-      hawkular_security_protocol = ""
+      hawkular_security_protocol = security_protocol_default
```

Comment:
Great catch!  Re-testing:

- [x] editing a single-endpoint (pre-darga) provider with security_protocol == nil, both show SSL
- [ ] editing a two-endpoint provider with security_protocol == nil
  - [x] Default shows SSL
  - [ ] **Hawkular shows `<Choose>`**
- [x] editing a two-endpoint provider with protocol already set values, both show appropriate value

I think it's because further down L170-L176, if the endpoint exists, we take its protocol.
Trying a fix...